packages/macro/src/index.macro.ts

```diff
@@ -12,7 +12,18 @@ function addImportStatement(importName: string, importPath: string, isDefault: b
   }
 
   const existingImportDeclaration = programPath.node.body.find(
-    p => t.isImportDeclaration(p) && p.source.value === importPath
+    p => {
+      // return t.isImportDeclaration(p) && p.source.value === importPath;
+      if (!t.isImportDeclaration(p) || p.source.value !== importPath) {
+        return false;
+      }
+
+      if (isDefault) {
```

Comment:
It ensures that the import statement exists and imports the exact element I want to import. **It checks both the import path and the import name**. It fixes the problem when I try to make the plugin import `useMemo` when react is already imported.